pkgs/development/python-modules/nltk/default.nix

```
@@ -22,14 +22,14 @@
 
 buildPythonPackage rec {
   pname = "nltk";
-  version = "3.9.1";
+  version = "3.9.2";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and use `build-system`